da/grpc/mockserv/cmd/main.go

```diff
@@ -4,15 +4,18 @@ import (
 	"flag"
 	"log"
 	"net"
+	"os"
 	"strconv"
 
 	grpcda "github.com/celestiaorg/rollmint/da/grpc"
 	"github.com/celestiaorg/rollmint/da/grpc/mockserv"
 	"github.com/celestiaorg/rollmint/store"
+	tmlog "github.com/tendermint/tendermint/libs/log"
```

Comment:
This is mixing "local" and 3rd party imports. `tmlog` should be imported in separate "section" before all the rollmint imports.

This should be detected by linter.